features/step_definitions/basic_steps.rb

```diff
@@ -12,6 +12,22 @@
     visit new_user_registration_path
 end
 
+When('I fill in {string} field with {string}') do |field, input|
+    fill_in field, with: input
+end
+
 When('I click {string}') do |element|
     click_on element
-end
\ No newline at end of file
+end
+
+When('I choose {string} from {string}') do |input, field|
+    select input, from: field
+end
+
+Given('I am logged in') do
+    visit new_user_session_path
+    fill_in 'Email', with: 'real@email.com'
+    fill_in 'Password', with: 'password'
+    click_on 'Log in'
```

Comment:
You can use a warden helper here 
```
Given(/^I am logged in as "([^"]*)"$/) do |name|
  user = User.find_by(name: name)
  login_as(user, scope: :user)
end
```
Check this page for more information
https://class.craftacademy.co/courses/course-v1:CraftAcademy+CA-CC-01+2018/courseware/2c610fa3ac07421aa323ea6bbb010083/19ff5d7b9ba84a279f1d448fba0b1653/?activate_block_id=block-v1%3ACraftAcademy%2BCA-CC-01%2B2018%2Btype%40sequential%2Bblock%4019ff5d7b9ba84a279f1d448fba0b1653